avar2.md

```diff
@@ -36,7 +36,22 @@ Delta values are in the range [-2,2). However, in the `ItemVariationData` they a
 
 # Processing
 
-Processing of axis values in a `avar` version 2 table starts off the same as a `avar` version 1 table. That is, axis coordinates are normalized and then mapped according to `axisSegmentMaps`.
+Processing of an axis value in a `avar` version 2 table starts off the same as in an `avar` version 1 table. That is, axis coordinates are normalized and then mapped according to `axisSegmentMaps`. Then the `avar` version 2 algorithm is applied to obtain the new axis value. Thus we have three steps:
+
+1. Initial normalization to convert user values to normalized values in the range [-1,1].
+2. Remapping of normalized values under `avar` version 1 to new values also in the range [-1,1].
+3. Remapping of normalized values under `avar` version 2 to new values clamped to the range [-1,1].
+
+The value obtained in step 3 is the final normalized value for the axis, and is to be used in the standard variation process described in [Algorithm for Interpolation of Instance Values](https://docs.microsoft.com/en-us/typography/opentype/spec/otvaroverview#algorithm-for-interpolation-of-instance-values) to obtain scalars for delta sets.
+
+
+
+Example, assuming an axis with minimum 300, default 400 and maxmium 700, and with an avar table that has v1 mappings of -1 to -1, 0 to 0, 0.5 to 0.4, 1 to 1:
```

Comment:
Can we move examples to a separate Examples section after the normative parts of the spec please?

Reply:
No problem. I had been thinking the same.

Reply:
I’ll also move "Inverse avar processing" section.